aten/src/ATen/native/cuda/SoftMax.cu

```diff
@@ -172,6 +173,19 @@ inline dim3 SoftMax_getBlockSize(int ILP, uint64_t dim_size) {
   return dim3(block_size);
 }
 
+inline dim3 SoftMaxForward_getBlockSize(uint64_t dim_size) {
+  uint64_t block_size = 1;
+  uint64_t max_block_size = std::min(dim_size / ILP, static_cast<uint64_t>(max_threads));
```

Comment:
looks like the build is breaking because ILP isn't defined here (was it intended to be an argument)?

Reply:
Whops. Wrong copy paste from my dev machine. I'll spend some time this week to make the PR landable and make sure all the tests pass.